tier4_planning_msgs/msg/VelocityLimitConstraints.msg

```diff
@@ -1,3 +1,4 @@
+float32 max_acceleration
```

Comment:
If possible, could you please add a comment here indicating that this is the maximum signed acceleration?

Reply:
Fixed in 1377eef427a564e0e8e3fe07ed17a749b1c58ad9